packages/scaffold/src/partials/w3m-header/index.ts

```diff
@@ -58,7 +59,12 @@ function headings() {
     SwapPreview: 'Preview swap',
     WalletSend: 'Send',
     WalletSendPreview: 'Review send',
-    WalletSendSelectToken: 'Select Token'
+    WalletSendSelectToken: 'Select Token',
+    ConnectWallets: 'Connect wallet',
+    ConnectSocials: 'All socials',
+    ConnectingSocial: AccountController.state.socialProvider
+      ? AccountController.state.socialProvider
+      : 'Connect Social'
```

Comment:
`Connect with Social`? Not convinced though

Reply:
This is according to design, we can change if needed